Makefile

```diff
@@ -176,43 +155,6 @@ kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 $(KUSTOMIZE): $(LOCALBIN)
 	test -s $(LOCALBIN)/kustomize || curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s $(KUSTOMIZE_VERSION) $(LOCALBIN)
 
-# go-get-tool will 'go get' any package $2 and install it to $1.
-PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
-define go-get-tool
-@[ -f $(1) ] || { \
-set -e ;\
-TMP_DIR=$$(mktemp -d) ;\
-cd $$TMP_DIR ;\
-go mod init tmp ;\
-echo "Downloading $(2)" ;\
-GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
-rm -rf $$TMP_DIR ;\
-}
-endef
-
-.PHONY: bundle
-bundle: manifests setup kustomize ## Generate bundle manifests and metadata, then validate generated files.
-	scripts/update-sample.sh
```

Comment:
I believe this was removed accidentally. This line was added by PR https://github.com/OpenLiberty/open-liberty-operator/pull/375/files and should be retained.

Reply:
@halim-lee I referred to this line in particular: `scripts/update-sample.sh` which is not present in the make target `bundle` ([here](https://github.com/OpenLiberty/open-liberty-operator/blob/58206329f707d74385c3b1357d3d8c4de484e206/Makefile#L202)). Could it be that it's not pushed to remote?

Reply:
Oh no - it's added back in!